### PR TITLE
Feat/usability updates

### PR DESCRIPTION
**Pull request by ludomikula**
## Proposed changes
- make default biz thresholds configurable via application.yaml and environment properties
- add possibility to disable certain endpoints via configuration (eg. overriding biz thresholds using API)
- add OpenAPI specification and a web UI to browse it

## Types of changes
What types of changes does your code introduce to Lowcoder?  
_Put an `x` in the boxes that apply._

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist
You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! 
This is simply a reminder of what we are going to look for before merging your code.  
_Put an `x` in the boxes that apply._

- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments
n/a

### CODE REVIEW

File: deploy/docker/README.md [hunk 1]
```diff
@@ -21,24 +21,29 @@ DOCKER_BUILDKIT=1 docker build -f deploy/docker/Dockerfile -t lowcoderorg/lowcod
...
+| `PUID`                          | ID of user running services. It will own all created logs and data. | `9001`                                                |
+| `PGID`                          | ID of group of the user running services.                           | `9001`                                                |
 | `MONGODB_URI`                   | Mongo database connection string                                    | `mongodb://localhost:27017/lowcoder?authSource=admin` |
-| `REDIS_URL`                     | Redis server URL                                                    | `redis://localhost:6379`                                |
-| `JS_EXECUTOR_URI`               | Node service URL                                                    | `http://localhost:6060`                                 |
-| `ENABLE_USER_SIGN_UP`           | Enable registration of new users                                    | `true`                                                  |
+| `REDIS_URL`                     | Redis server URL                                                    | `redis://localhost:6379`                              |
```
Comment (FalkWolsky) [inline deploy/docker/README.md:34]: We could rename it to "Redis database connection string" - to stay consistent..

File: deploy/docker/README.md [hunk 2]
```diff
@@ -21,24 +21,29 @@ DOCKER_BUILDKIT=1 docker build -f deploy/docker/Dockerfile -t lowcoderorg/lowcod
...
 | `ENCRYPTION_SALT`               | Salt used for encrypting password                                   | `lowcoder.org`                                        |
-| `CORS_ALLOWED_DOMAINS`          | CORS allowed domains                                                | `*`                                                     |
-| `LOWCODER_API_SERVICE_URL`    | Lowcoder API service URL                                          | `http://localhost:8080`                                 |
-| `LOWCODER_NODE_SERVICE_URL`   | Lowcoder Node service (js executor) URL                           | `http://localhost:6060`                                 |
+| `CORS_ALLOWED_DOMAINS`          | CORS allowed domains                                                | `*`                                                   |
+| `LOWCODER_API_SERVICE_URL`      | Lowcoder API service URL                                            | `http://localhost:8080`                               |
+| `LOWCODER_NODE_SERVICE_URL`     | Lowcoder Node service (js executor) URL                             | `http://localhost:6060`                               |
```
Comment (FalkWolsky) [inline deploy/docker/README.md:41]: How is this different from "JS_EXECUTOR_URI"?

File: deploy/docker/README.md [hunk 3]
```diff
@@ -21,24 +21,29 @@ DOCKER_BUILDKIT=1 docker build -f deploy/docker/Dockerfile -t lowcoderorg/lowcod
...
+| `MONGODB_ENABLED`               | If **true** mongo database is started in the container              | `true`                                                |
+| `API_SERVICE_ENABLED`           | If **true** lowcoder api-service is started in the container        | `true`                                                |
+| `NODE_SERVICE_ENABLED`          | If **true** lowcoder node-service is started in the container       | `true`                                                |
+| `FRONTEND_ENABLED`              | If **true** lowcoder web frontend is started in the container       | `true`                                                |
+| `PUID`                          | ID of user running services. It will own all created logs and data. | `9001`                                                |
+| `PGID`                          | ID of group of the user running services.                           | `9001`                                                |
 | `MONGODB_URI`                   | Mongo database connection string                                    | `mongodb://localhost:27017/lowcoder?authSource=admin` |
```
Comment (FalkWolsky) [inline deploy/docker/README.md:33]: Lets name it Consistent MONGODB_URL

**Review by FalkWolsky — COMMENTED** [review]
Just the 2 points about ENV VAR Names as we briefly discussed.

File: deploy/docker/README.md (hunk 2 above)
Comment (ludomikula) [reply deploy/docker/README.md:40]: Agreed. We'll keep LOWCODER_NODE_SERVICE_URL

File: deploy/docker/README.md (hunk 3 above)
Comment (ludomikula) [reply deploy/docker/README.md]: Agreed.

**Review by FalkWolsky — APPROVED** [review]
Thank you very much for the cleanup!

**Pull request merged**